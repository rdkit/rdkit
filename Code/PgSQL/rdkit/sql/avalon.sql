select tanimoto_sml(avalon_fp('c1ccccc1'::mol),avalon_fp('c1ccccc1'::mol));
select tanimoto_sml(avalon_fp('c1ccccc1'::mol),avalon_fp('c1ccccc1'::mol,true));
select tanimoto_sml(avalon_fp('c1ccccc1'::mol),avalon_fp('c1ccccn1'::mol));
