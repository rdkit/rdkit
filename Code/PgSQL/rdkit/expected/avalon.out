select tanimoto_sml(avalon_fp('c1ccccc1'::mol),avalon_fp('c1ccccc1'::mol));
 tanimoto_sml 
--------------
            1
(1 row)

select tanimoto_sml(avalon_fp('c1ccccc1'::mol),avalon_fp('c1ccccc1'::mol,true));
 tanimoto_sml 
--------------
         0.25
(1 row)

select tanimoto_sml(avalon_fp('c1ccccc1'::mol),avalon_fp('c1ccccn1'::mol));
   tanimoto_sml    
-------------------
 0.333333333333333
(1 row)

